test/node_modules/duplicate_symbols/binding.gyp

```diff
@@ -0,0 +1,19 @@
+{
+  "target_defaults": {
+    "include_dirs": [
+      "<!(node -e \"require('nan')\")"
+    ],
```

Comment:
This can go? The add-on doesn't use nan (doesn't even depend on it in package.json.)

Reply:
Actually, it does use nan. I added the dependency to package.json.